src/browser/components/MattermostView.jsx

```diff
@@ -157,7 +157,16 @@ const MattermostView = createReactClass({
         console.warn(message);
         break;
       case 2:
-        console.error(message);
+        const fileURL = this.extractFileURL(e.message);
```

Comment:
https://eslint.org/docs/rules/no-case-declarations 
Please take care about indent when fixing this.